models/intel/horizontal-text-detection-0001/accuracy-check.yml

```
@@ -0,0 +1,47 @@
+models:
+  - name: horizontal-text-detection-0001
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/horizontal-text-detection-0001/FP32/horizontal-text-detection-0001.xml
+        weights: intel/horizontal-text-detection-0001/FP32/horizontal-text-detection-0001.bin
+        adapter:
+            type: class_agnostic_detection
+            scale: 0.0014204
```

please fix linting error:
`
16:58:24    11:13     error    wrong indentation: expected 10 but found 12  (indentation)`
```suggestion
          type: class_agnostic_detection
          scale: 0.0014204
```